components/Table/Table.tsx

```diff
@@ -19,7 +19,6 @@ export const Tbody = styled('tbody', {
 export const Th = styled('th', Label, {
   // override Label
   display: 'table-cell',
-  verticalAlign: 'inherit',
```

Comment:
This seems to be reverting this PR:
- https://github.com/traefik/faency/pull/331

Is that intentional?

Reply:
It was totally unintentional :P